ci/src/datakit_ci.mli

```diff
@@ -0,0 +1,664 @@
+(** Datakit CI
```

Comment:
Didn't we decide to use `DataKit` rather than `Datakit` at some point? Also, it's hard to comment on changes to the API now they appear as a new file.